sdk/java-sdk-testkit/src/main/java/kalix/javasdk/testkit/EventSourcedResult.java

```diff
@@ -77,7 +77,7 @@
    *
    * @return The next event if it is of type E, for additional assertions.
    */
-  <E> E getNextEventOfType(Class<E> expectedClass);
+  <T> T getNextEventOfType(Class<T> expectedClass);
```

Comment:
I agree that we should not type the events in the result because it will be a breaking change at users face.

Reply:
Btw, just noticed this change was a leftover from when I tried to type this class as well. I reverted this as well now (so it's unchanged and consistent with the comment).